.matrix.yml

```diff
@@ -3,6 +3,44 @@ JENKINS_TESTTYPES:
   - univention-bareos
 
 OS:
+  Sanitize:
+    "F37":
+      TYPE: scripted
+      IMAGE: fedora37
+      BUILD_SCRIPT: CD/sanitize/build-sanitize.sh
```

Comment:
we can move that to the Bareos repository, you just use "bareos/..." as a path here.
The script path is relative to Jenkins' $WORKSPACE and it always run with cwd set to the top of the cloned bareos repository.
So you can test and develop the script locally and it should just work in Jenkins.

Reply:
done